SPECS/openssl/openssl-1.1.1-load-default-engines.patch

```diff
@@ -1,20 +1,3 @@
-diff --git a/crypto/engine/eng_all.c b/crypto/engine/eng_all.c
-index b675ed7892..b7abf3d61f 100644
---- a/crypto/engine/eng_all.c
-+++ b/crypto/engine/eng_all.c
-@@ -10,6 +10,12 @@
- #include "internal/cryptlib.h"
- #include "eng_local.h"
- 
-+ __attribute__((constructor))
```

Comment:
Since we already had a few successful package builds, it will be safer to also increment the release in the spec file and add a changelog entry.